docs/APIBaselines.md

```diff
@@ -6,8 +6,24 @@ This document contains information regarding API baseline files and how to work
 
 When creating a new implementation (i.e. src) project, it's necessary to manually add API baseline files since API baseline are enabled by default. If the project is a non-shipping or test only project, add `<AddPublicApiAnalyzers>false</AddPublicApiAnalyzers>` to the project to disable these checks. To add baseline files to the project:
 
-1. Copy eng\PublicAPI.empty.txt to the project directory and rename it to PublicAPI.Shipped.txt
-1. Copy eng\PublicAPI.empty.txt to the project directory and rename it to PublicAPI.Unshipped.txt
+1. `cp .\eng\PublicAPI.empty.txt {new folder}\PublicAPI.Shipped.txt`
+1. `cp .\eng\PublicAPI.empty.txt {new folder}\PublicAPI.Unshipped.txt`
+1. Update AspNetCore.sln and relevant `*.slnf` file to include the new project
+1. `{directory containing relevant *.slnf}\startvs.cmd`
+1. F6 *or whatever your favourite build gesture is*
+1. Click on a RS0016 (or whatever) error
+1. Right click in editor on underscored symbol or go straight to the “quick fix” icon to its left. Control-. also works.
+1. Choose “Add Blah to public API” / “Fix all occurrences in … Solution”
+1. Click Apply
+1. F6 *again to see if the fixer missed anything or if other RS00xx errors show up (not uncommon)*
+1. Suppress or fix other problems as needed but please suppress (if suppressing) using attributes and not globally or with `#pragma`s because attributes make the justification obvious e.g. for common errors that can't be fixed
+    `[SuppressMessage("ApiDesign", "RS0026:Do not add multiple public overloads with optional parameters", Justification = "Required to maintain compatibility")]`
+    or
+    `[SuppressMessage("ApiDesign", "RS0027:Public API with optional parameter(s) should have the most parameters amongst its public overloads.", Justification = "Required to maintain compatibility")]`
+
+More information available in
+-	https://github.com/dotnet/roslyn-analyzers/blob/master/src/PublicApiAnalyzers/PublicApiAnalyzers.Help.md
+-	https://github.com/dotnet/roslyn-analyzers/blob/master/src/PublicApiAnalyzers/Microsoft.CodeAnalysis.PublicApiAnalyzers.md
```

Comment:
Suggest adding a note to start at step (4) when making changes in projects with existing API baselines.

Reply:
We probably need at least a note about the `*REMOVED*` prefix whether or not the fixer will handle that automatically. Doubt other syntax notes are interesting to most people.

Speaking of special cases 😺 @BrennanConroy did the fixer handle most / all of the PublicAPI.Unshipped.txt changes in #29219 without manual changes, aside from any `[SuppressMessage(...)]` attributes you needed❔

Reply:
No, I gave up on the fixer and did it all manually

Reply:
Wow, I remember the fixer doing a pretty good job even when fixing all instances of a problem in an entire (albeit filtered) solution. What went wrong e.g. is there an issue or three we should report in https://github.com/dotnet/roslyn-analyzers❔

Reply:
It seemed inconsistent about which file it edited, so I couldn't just blindly accept whatever changes it wanted to make.

Reply:
Ah, for example does it edit PublicAPI.Shipped.txt files rather than add `*REMOVED*` lines in PublicAPI.Unshipped.txt❔

@sharwell and @mavasani is that a known issue. Could the fixer be fixed to _always_ leave PublicAPI.Shipped.txt alone❔ (Well, except if you add a gesture for use after a release -- where the files should be merged.)